test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,28 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertHaveNull(Object... objects) { assertTrue(Utils.isAnyNull(objects)); }
+
+    private void assertNoNull(Object... objects) { assertFalse(Utils.isAnyNull(objects)); }
+
+    @Test
+    public void testIsAnyNull() throws Exception {
```

Comment:
Also, I think it's better to put all `@Test` functions together on top and helper functions at the end of the file.